frame/lending/src/tests/vault.rs

```diff
@@ -67,3 +75,115 @@ fn test_vault_market_can_withdraw() {
 		);
 	});
 }
+
+// Generates well funded accounts
+fn generate_accounts(amount: u128) -> Vec<AccountId> {
+	let mut accounts = Vec::new();
+	let pid = PalletId(*b"markttst");
```

Comment:
you do not need to derive acounts from pallet id,

Reply:
it will be simpler to bump +1 existing accounts to get next

pub static CHARLIE: Lazy<AccountId> = Lazy::new(|| {
	AccountId::from_raw(hex!("0000000000000000000000000000000000000000000000000000000000000002"))
});

Reply:
@dzmitry-lahoda to bump existing account we have to bump value in string or in [u8;32] which is kind of tricky. into_sub_account() do it for us. I have changed generate_accounts() implementation to remove "pallet" references

Reply:
you may have u32 number, serde it and make trailing zeroes. other pallets do that. `trailing` key word.

Reply:
Done